tests/test_live.py

```diff
@@ -87,7 +87,7 @@ def test_instruments():
         host=get_platform_dispatcher(),
         instrument="mock",
     )
-    assert disp.get_instruments_list() == ['magic', 'isgri', 'jemx', 'polar', 'antares', 'spi_acs']
+    assert disp.get_instruments_list() == ['isgri', 'jemx', 'polar', 'antares', 'spi_acs']
```

Comment:
This test, now passes

Reply:
any ideas @volodymyrss  ?

Reply:
what is the problem precisely?

Reply:
the list of instruments returned does not includ `magic`

Reply:
it should not. we do not provide magic.